src/main/java/org/broadinstitute/hellbender/engine/GATKTool.java

```diff
@@ -250,7 +249,20 @@ public SAMSequenceDictionary getReferenceDictionary() {
      * @return best available sequence dictionary given our inputs
      */
     public SAMSequenceDictionary getBestAvailableSequenceDictionary() {
-        return reference != null ? reference.getSequenceDictionary() : (reads != null ? reads.getSequenceDictionary() : null);
+        if (hasReference()){
+            reference.getSequenceDictionary();
+        }
+        if (hasReads()){
+            return reads.getSequenceDictionary();
+        }
+        if (hasFeatures()){
+            final List<SAMSequenceDictionary> dictionaries = features.getSequenceDictionaries();
+            //If there is just one, it clearly is the best. Otherwise, noone is best.
+            if (dictionaries.size() == 1){
+                return dictionaries.get(0);
+            }
```

Comment:
just seems to be too much pain right now. We can do it if there's a usecase for it